backend/app/adapter/db/changelog.go

```diff
@@ -33,7 +33,7 @@ FROM "%s";`,
 		return []entity.Change{}, err
 	}
 
-	var changeLog []entity.Change
+	changeLog := []entity.Change{}
```

Comment:
Let's add a comment on why this needed in the code.

Reply:
@byliuyang  Should i change expectedChangeLog in testCase to []entity.Change{nil} instead ?
https://stackoverflow.com/questions/29164375/correct-way-to-initialize-empty-slice

Reply:
That does'nt work...

Reply:
Comment should explain why instead of what. Everyone knows `what`. They don't know `why`.

Reply:
@alldroll Agree!

Reply:
the consumer of GetChangeLog expects empty slice instead of `nil` if there are no records